src/js/features/search/makeGetSearchFields.thunk.ts

```diff
@@ -0,0 +1,15 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { searchFieldsUrl } from '@/constants/configConstants';
+import { printAPIError } from '@/utils/error.util';
+import { SearchFieldResponse } from '@/types/search';
+
+export const makeGetSearchFields = createAsyncThunk<SearchFieldResponse, void, { rejectValue: string }>(
+  'query/makeGetSearchFields',
+  async (_, { rejectWithValue }) => {
```

Comment:
this function isnt actually async
plus could be written as an expression function rather than with `return`